board/aspeed/evb_ast2700/evb_ast2700.c

```diff
@@ -325,6 +330,36 @@ int set_board_info(const u8* scm_eeprom_buf, const u8* hpm_eeprom_buf)
 	return 0;
 }
 
+void mount_safs_spi_mux()
+{
+	printf("configuring gpios for eDAF ...\n");
+	/* mount P0 BIOS SPI */
+	run_command("gpio set 169", 0); //GPIO V1
+	run_command("gpio set 170", 0); //GPIO V2
+	printf("configuring gpios for eDAF ...Done\n");
+}
+
+void train_ltpi(int retry)
+{
+        // set espi GPIO strap to low and train ltpi
+        run_command("mw 14c02404 0", 0);
+        run_command("gpio clear 10", 0);
```

Comment:
int run_command(const char *cmd, int flag); - please handle return , will help for debug incase of failure.